cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -134,6 +137,22 @@
                     );
                 });
 
+                // prevents single click from messing up the edit call
```

Comment:
@dhorelik @FinalAngel either way, we're screwed... I'm ignorant about frontend, but how about an overlay on top of the link which "steals" the events in content mode/draft?